### PR TITLE
Resources fixes

### LINKED ISSUE #112: Resources page bugs fixes
fix filters bugs and make responsive

### PR DESCRIPTION
**Pull request by fadime-ozdemir**
-fix bugs with filters
-make responsiveness of resources page, header, and footer
-fix browse categories

### CODE REVIEW

File: src/components/NgoDetails/index.jsx [hunk 1]
```diff
@@ -2,19 +2,28 @@ import React from "react";
...
 import "./index.css";
 import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
 
 const NgoDetails = ({ details }) => {
   const { t } = useTranslation();
-
+  const initialLanguage = useSelector((state) => state.languageReducer);
+  const renderDetails = () => {
+    console.log(initialLanguage);
```
Comment (halitbatur) [inline src/components/NgoDetails/index.jsx]: console.log

File: src/components/NgoDetails/index.jsx [hunk 2]
```diff
@@ -2,19 +2,28 @@ import React from "react";
...
+    switch (initialLanguage) {
+      case "ar":
+        return details.serviceDescription.arabic;
```
Comment (halitbatur) [inline src/components/NgoDetails/index.jsx:14]: these won't be needed I think cause the i18n automaticaly transaltes it. all u have to do is have details.serviceDescription in the 3 json files and it changes it auto

File: src/components/NgoDetails/index.jsx (hunk 1 above)
Comment (khaldoon9kh) [inline src/components/NgoDetails/index.jsx]: delete consol.

File: src/components/NgoDetails/index.jsx [hunk 3]
```diff
@@ -2,19 +2,28 @@ import React from "react";
 import { Button } from "react-bootstrap";
 import "./index.css";
 import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
 
 const NgoDetails = ({ details }) => {
   const { t } = useTranslation();
-
+  const initialLanguage = useSelector((state) => state.languageReducer);
```
Comment (khaldoon9kh) [inline src/components/NgoDetails/index.jsx:9]: Aha! now I see how you do it!

File: src/components/NgoDetails/index.jsx (hunk 2 above)
Comment (halitbatur) [reply src/components/NgoDetails/index.jsx:14]: by saying 3 json files i mean each one for the ar eng or tr

File: src/redux/action/index.jsx [hunk 4]
```diff
@@ -13,3 +13,10 @@ export const wordsSearchedAction = (data) => {
     payload: data,
   };
 };
+
+export const languageAction = (data) => {
+  return {
+    type: LANGUAGE,
+    payload: data,
+  };
+};
```
Comment (khaldoon9kh) [inline src/redux/action/index.jsx:22]: SMART!

File: src/components/NgoDetails/index.jsx (hunk 2 above)
Comment (halitbatur) [reply src/components/NgoDetails/index.jsx:14]: are the ngo details coming for the json files in i18n or is it fetched with firebase ?

**Review by khaldoon9kh — CHANGES_REQUESTED** [review]
LGTM.
However, I think you made this branch from "Fix Title" branch and it will make some conflicts I think.
Amazing work on the language switcher!

File: src/components/NgoDetails/index.jsx (hunk 2 above)
Comment (halitbatur) [reply src/components/NgoDetails/index.jsx:14]: oh okai its coming from firebase, ignore what i just said.

**Review by halitbatur — APPROVED** [review]

File: src/components/NgoDetails/index.jsx (hunk 2 above)
Comment (fadime-ozdemir) [reply src/components/NgoDetails/index.jsx:14]: It comes from the database not from the translation

**Comment by fadime-ozdemir** [comment]
PTAL

**Review by khaldoon9kh — APPROVED** [review]
LGTM

**Pull request merged**